charts/aws-fsx-csi-driver/templates/csidriver.yaml

```diff
@@ -3,4 +3,12 @@ kind: CSIDriver
 metadata:
   name: fsx.csi.aws.com
 spec:
-  attachRequired: false
+  attachRequired: {{ .Values.csidriver.attachRequired }}
```

Comment:
Most of these settings only define properties of the CSI driver. I don't think it makes sense to have these configurable